### ISSUE TITLE
Scraper selections for movies not followed?

### ISSUE THREAD

**Issue by wkearney99** [adultmm/AdultMediaManager#73]
I've noticed it will sometimes still try to use it even if GayEroticVideoIndex.com is de-selected in the scraper list.   Doesn't seem to happen on all movies, only ones not found with other scrapers first, and then only if there's something in iafd that might indicate it.  still, if the scraper box is unchecked then it shouldn't go there, at all.

**Comment by wkearney99** [comment]
I noticed this is also true for Jeedo.  Even with the box unchecked it will still throw errors.  It's like the checkbox option is being ignored.  This with the scraper disabled:

```
2022-12-19 15:22:23.2154,scraper.Image.AdultScraper.AdultScraper_Image,scraper.Image.AdultScraper.AdultScraper_Image.Scraper_Movie,5,TRACE,[AdultScraper_Image] [Scraper_Movie] [Start],
2022-12-19 15:22:29.3277,AdultScraper.Shared.ErrorReporter,AdultScraper.Shared.ErrorReporter+MessageBoxErrorReporter.ReportError,5,ERROR,An error occurred during downloading the movie information.  Scraper: Jeedoo Message: The request was aborted: The connection was closed unexpectedly.,"EXCEPTION OCCURRED:System.Net.WebException: The request was aborted: The connection was closed unexpectedly.
   at System.Net.HttpWebRequest.GetResponse()
   at HtmlAgilityPack.HtmlWeb.Get(Uri uri, String method, String path, HtmlDocument doc, IWebProxy proxy, ICredentials creds)
   at HtmlAgilityPack.HtmlWeb.LoadUrl(Uri uri, String method, WebProxy proxy, NetworkCredential creds)
   at HtmlAgilityPack.HtmlWeb.Load(Uri uri, String method)
   at HtmlAgilityPack.HtmlWeb.Load(String url)
   at AdultScraper.Shared.JeedooScraper.GetByIdInternal(String externalId, String lang)
   at AdultScraper.Shared.MovieScraper.GetById(String externalId, String lang)*   at System.Net.HttpWebRequest.GetResponse()
   at HtmlAgilityPack.HtmlWeb.Get(Uri uri, String method, String path, HtmlDocument doc, IWebProxy proxy, ICredentials creds)
   at HtmlAgilityPack.HtmlWeb.LoadUrl(Uri uri, String method, WebProxy proxy, NetworkCredential creds)
   at HtmlAgilityPack.HtmlWeb.Load(Uri uri, String method)
   at HtmlAgilityPack.HtmlWeb.Load(String url)
   at AdultScraper.Shared.JeedooScraper.GetByIdInternal(String externalId, String lang)
   at AdultScraper.Shared.MovieScraper.GetById(String externalId, String lang)"

```
The big downside here is it causes the program to exit after dismissing the error dialog.

**Comment by wkearney99** [comment]
To follow up, this seems to be happening AFTER a data scrape has been made and then it crashes out when trying to do an image scrape.  The checkbox in data scraping does not correspond to the scrapers used for images, apparently?

**Comment by adultmm** [comment]
Thanks for the error report. 

You're right, those checkboxes are not applied during the image scraping stage. However, we are using scrapers only where there is movieId selected (and stored in the database) in an earlier data scraping, or an exact match exists by title and year. 

For the time being there's no Movie selection dialogue during the image scraping. 

In your case, there is a JeeDoo id for the movie, so AMM tries to download the movie page, and it causes some network error. But it's a handled error, so the crash is not caused by this error, unfortunatelly it'll be somtehing else. 

Could you send me a bigger part (or full) of the log to try to reproduce this error?

**Comment by adultmm** [comment]
Fixed in [v0.12.0-BETA.2](https://github.com/adultmm/AdultMediaManager/releases/tag/v0.12.0-BETA.2). And you can also enable the used scrapers for Data/Image scraping separately.
